docs_src/security/tutorial003_py310.py

```diff
@@ -66,19 +66,30 @@ async def get_current_user(token: str = Depends(oauth2_scheme)):
 
 async def get_current_active_user(current_user: User = Depends(get_current_user)):
     if current_user.disabled:
-        raise HTTPException(status_code=400, detail="Inactive user")
+        raise HTTPException(
```

Comment:
I would disagree here.

400 definition:
`The server cannot or will not process the request due to something that is perceived to be a client error (e.g., malformed request syntax, invalid request message framing, or deceptive request routing).` That is not the case here, it is not a client error.

I would reckon that 403 is more appropriate, especially because the authentication passed but the user is not allowed: `The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401 Unauthorized, the client's identity is known to the server.`

Reply:
I agree with you